acceptance/gossip_peerings_test.go

```diff
@@ -45,11 +44,6 @@ func checkGossip(t *testing.T, l *localcluster.Cluster, d time.Duration,
 		case <-stopper:
 			t.Fatalf("interrupted")
 			return nil
-		case e := <-l.Events:
-			if log.V(1) {
-				log.Infof("%+v", e)
-			}
-			return fmt.Errorf("event: %+v", e)
```

Comment:
I'd like to see the `AssertAndStop` functionality folded into `Stop`. That is, `Stop` always drains the events buffer looking for unexpected events. Tests which manually restart nodes should do the draining and checking themselves. This would be enforced by `Stop` causing the test to fail it the event buffer has not been drained and contains unexpected events.